From 2c66d6c50255c20f5243812b9919fa8bab90b02c Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 17 Nov 2015 20:14:31 +0000
Subject: [PATCH 195/462] MANTIS 0037350 extension: integration of EZ-Access
 device for KIS kiosk

added LIB_KEYBOARD_SERVICE =  $(_LIB)\ipsKeyboardDriverService.lib



git-svn-id: svn://localhost/SelfServiceBuild/trunk@297 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.common.mak | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SelfServiceBuild/Build/cpp/makefile.common.mak b/SelfServiceBuild/Build/cpp/makefile.common.mak
index be01f36..c86a70c 100644
--- a/SelfServiceBuild/Build/cpp/makefile.common.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.common.mak
@@ -358,6 +358,7 @@ LIB_DST_DEVICESCOMMON  = $(_LIB)\devicescommon.lib
 LIB_DST_SERIALCON      = $(_LIB)\mserialcon.lib
 LIB_DST_TERMINAL       = $(_LIB)\mterminal.lib
 LIB_DST_GPPDEV_CUSTOM  = $(_LIB)\GPPDevCustom.lib
+LIB_DST_KEYBOARD_SERVICE= $(_LIB)\ipsKeyboardDriverService.lib
 
 DLL_MASSAIJNI          = $(_DLL)\massaijni.dll
 
@@ -372,6 +373,7 @@ LIB_DEVICESCOMMON      = $(LIB_DST_DEVICESCOMMON)
 LIB_SERIALCON          = $(LIB_DST_SERIALCON) 
 LIB_TERMINAL           = $(LIB_DST_TERMINAL) 
 LIB_GPPDEV_CUSTOM      = $(LIB_DST_GPPDEV_CUSTOM)
+LIB_KEYBOARD_SERVICE   = $(LIB_DST_KEYBOARD_SERVICE)
 
 
 ######################################################################
-- 
2.41.0.windows.1

